java/bogdana_maksiuta/src/com/zoolatech/lecture2/tasks/_2/Robot.java

```diff
@@ -0,0 +1,189 @@
+package com.zoolatech.lecture2.tasks._2;
+
+/**
+ * Please carefully read the task description: you need to have a class that provides following methods:
+ *
+ * move()
+ * move(int tiles)
+ * turnLeft()
+ * turnRight()
+ * currentLocation()
+ * currentDirection()
+ * constructors to properly create an object of this class
+ */
+public class Robot {
+    private int h = 0;
+    private int w = 0;
+    private String direction = "south";
+    private int widthRoom;
+    private int heightRoom;
+    public int getH() {
+        return h;
+    }
+    public void setH(int h) {
+        this.h = h;
+    }
+    public int getW() {
+        return w;
+    }
+    public void setW(int w) {
+        this.w = w;
+    }
+    public String getDirection() {
+        return direction;
+    }
+    public void setDirection(String direction) {
+        this.direction = direction;
+    }
+    public int getWidthRoom() {
+        return widthRoom;
+    }
+    public void setWidthRoom(int widthRoom) {
+        this.widthRoom = widthRoom;
+    }
+    public int getHeightRoom() {
+        return heightRoom;
+    }
+    public void setHeightRoom(int heightRoom) {
+        this.heightRoom = heightRoom;
+    }
+
+    public void move() {
+        switch (getDirection()) {
```

Comment:
same: you can access to instance variables **inside of a class** directly, without a getter method